src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -48,7 +48,8 @@ class StopPlayTest : public SimulatedErForceSimPlayTestFixture
              field.enemyDefenseArea().negXPosYCorner()});
 };
 
-// TODO (#2602): test failing since a robot isn't avoiding the ball
+// TODO (#2519): The ball is not an obstacle for robots running crease defense tactic,
+//               as a result, they collide with the ball and the test fails
```

Comment:
Why is the ball not being considered an obstacle for crease defender?

Reply:
I'm not fully sure why. It seems like it was initially added to the ignored motion constraint list (in `motion_constraint_visitor.cpp`) by @jonathanlew when he ported crease defense play over to FSMs in #2030.